boards/nordic/thingy53/board.c

```diff
@@ -7,30 +7,8 @@
 #include <zephyr/init.h>
 #include <zephyr/kernel.h>
 
-/* Initialization chain of Thingy:53 board requires some delays before on board sensors
- * could be accessed after power up. In particular bme680 and bmm150 sensors require,
- * respectively 2ms and 1ms power on delay. In order to avoid delays sum, common delay is
- * introduced in the board start up file. Below asserts ensure correct initialization order:
- * on board regulators, board init (this), sensors init.
- */
-#if !defined(CONFIG_TRUSTED_EXECUTION_SECURE)
-#if defined(CONFIG_REGULATOR_FIXED)
-BUILD_ASSERT(CONFIG_THINGY53_INIT_PRIORITY > CONFIG_REGULATOR_FIXED_INIT_PRIORITY,
-	"CONFIG_THINGY53_INIT_PRIORITY must be higher than CONFIG_REGULATOR_FIXED_INIT_PRIORITY");
```

Comment:
This will no longer be true and the delay will be applied before sensors are powered on.

Reply:
Removed